apps/src/dance/blockly/blocks.js

```diff
@@ -205,7 +205,7 @@ export default {
         // Must be marked EDITABLE so that cloned blocks share the same var name
         fieldLabel.EDITABLE = true;
         this.setHelpUrl(Blockly.Msg.VARIABLES_GET_HELPURL);
-        Blockly.cdoUtils.setHSV(this, 631, 0.84, 0.8);
+        Blockly.cdoUtils.handleColorAndStyle(this, [631, 0.84, 0.8]);
```

Comment:
add a constant for this color too?

Reply:
This one is interesting because the hue value is outside the expected range (0-360º), and as far as I can tell this code isn't actually used. Looking more closely, it's for behavior_get blocks, which don't actually exist in Dance. Further, I think it's a typo and was meant to be `136, 0.84, 0.8` which aligns with the hue used for behavior blocks in Sprite Lab. I can update.